src/services/transfers.service.ts

```diff
@@ -62,21 +61,21 @@ class TransfersService {
 
     const transferWithoutTheLastItem = transfers.slice(0, pageSize)
 
-    this.currentCursor = transferWithoutTheLastItem[transfers.length - 1]?.id
+    this.currentPage++
```

Comment:
Why would you need to increment a current page variable  on the transfer service instance and what is it even for?

Reply:
But why is it on the transfer service class this doesn't seem to be stateless as a service should be.

Reply:
Fixed. There is an issue on `TranferControllers` test regarding dependencies when the runner tries to run the test. This was fixed but something happened that again there is the issue with the exports. I will fix it tomorrow